src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -114,19 +116,50 @@ private IEnumerable<LibraryExport> ExportLibraries(Func<LibraryDescription, bool
                     analyzerReferences.AddRange(libraryExport.AnalyzerReferences);
                 }
 
-                yield return LibraryExportBuilder.Create(library)
+                var builder = LibraryExportBuilder.Create(library);
+                if (_runtime != null && _runtimeFallbacks != null)
+                {
+                    builder.WithRuntimeAssemblyGroups(TrimAssetGroups(libraryExport.RuntimeAssemblyGroups, _runtimeFallbacks));
```

Comment:
Actually, you are technically correct, but what this code does is trim items out of the `runtimeTargets` section of the lock file (which are only present in RID-less sections of the lock file). So while this code will hit on standalone apps, it will no-op since there will never be anything to trim (because the restore did all that necessary trimming)